newrelic-integration-e2e/pkg/agent/helper.go

```diff
@@ -70,11 +79,21 @@ func makeDirs(perm os.FileMode, dirs ...string) error {
 	return nil
 }
 
+func removeDirs(dirs ...string) error {
```

Comment:
which is the difference with removeDirectories?

Reply:
Actually It's not used